pkg/subctl/cmd/gather.go

```diff
@@ -190,13 +186,34 @@ func gatherDiscovery(dataType string, info *gather.Info) bool {
 }
 
 func gatherBroker(dataType string, info *gather.Info) bool {
+	brokerRestConfig, err := getBrokerRestConfig(info.Submariner)
+	exitOnError("Error getting Broker rest config", err)
+
+	dynClient, clientSet, err := getClients(brokerRestConfig)
+	exitOnError("Error getting client %s", err)
+
+	info = &gather.Info{
+		Clients:     gather.Clients{ClientSet:clientSet, DynClient:dynClient},
+		RestConfig:  brokerRestConfig,
+		ClusterName: "broker",
+	}
+
+	brokerNamespace := info.Submariner.Spec.BrokerK8sRemoteNamespace
+
 	switch dataType {
 	case Logs:
-		info.Status.QueueWarningMessage("No logs to gather on Broker")
+		info.Status.QueueSuccessMessage("No pods accessible. No logs to collect on Broker")
 	case Resources:
-		_, _ = getBrokerRestConfig(info.Submariner)
+        //NewBrokerClusterRole allows only below resources to be queried
+		gather.Endpoints(info, SubmarinerNamespace)
+
+		gather.Clusters(info, brokerNamespace)
+
+		//gather.EndpointSlices(info, brokerNamespace)
+
+		//gather.ServiceImports(info, brokerNamespace)
 
-		info.Status.QueueWarningMessage("Gather Broker Resources not implemented yet")
+		//gather.ServiceExports(info, brokerNamespace)
```

Comment:
Function implementation in PR #1196

Reply:
Is the ns "brokerNamespace" correct? I don't see any of these resources listed on my Broker.